CONTRIBUTING.md

```diff
@@ -29,7 +29,7 @@ docker run -v $(pwd):/src/grpc-gateway --rm jfbrandhorst/grpc-gateway-build-env
         make realclean && \
         make examples SWAGGER_CODEGEN="${SWAGGER_CODEGEN}"'
 docker run -itv $(pwd):/grpc-gateway -w /grpc-gateway --entrypoint /bin/bash --rm \
-    l.gcr.io/google/bazel -c 'bazel run :gazelle_fix; bazel run :buildifier'
+    l.gcr.io/google/bazel -c 'bazel run :gazelle fix; bazel run :buildifier'
```

Comment:
I believe this should just be `bazel run :gazelle`

Reply:
Ah ok, I'll update the CI configuration too